app/helpers/matrix_box_helper.rb

```diff
@@ -84,7 +84,8 @@ def matrix_box_details(presenter, object_id, identify)
       [
         matrix_box_what(presenter, object_id, identify),
         matrix_box_where(presenter),
-        matrix_box_when_who(presenter)
+        matrix_box_when_who(presenter),
+        matrix_box_source_credit(presenter)
```

Comment:
@nimmolo 
This plus `matrix_box_source_credit` result in including a source credit in indexes. (To me it's unexpected and strange for the Box to differ between Index and ActivityLog.)
- Will that substantially slow down the indexes?
- Does it make the indexes too noisy?